client/components/room/Room.jsx

```diff
@@ -1,41 +1,50 @@
-import io from 'socket.io-client';
 import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
 import React from 'react';
 import Video from './Video';
 import Workspace from './Workspace';
 import Chat from './Chat';
 
+import io from 'socket.io-client';
 const server = location.origin;
 const socket = io(server);
 
 class Room extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      redirect: false
+    };
   }
 
   componentDidMount() {
     socket.emit('join room', this.props.roomName);
+    socket.on('full', (room) => {
+      console.log(`${room} is full`);
+      this.setState({ redirect: true });
+    });
```

Comment:
Yeah,  I plan on writing an action and reducer for this, but just wanted to hack something together to show proof of concept for the redirect.